WordPress/WordPressUITests/Tests/LoginTests.swift

```diff
@@ -33,6 +33,40 @@ class LoginTests: XCTestCase {
         XCTAssert(welcomeScreen.isLoaded())
     }
 
+    func testEmailMagicLinkLogin() {
```

Comment:
@rachelmcr Are we planning for this test to actually test the login functionality via magic link? I'm not sure but I think we'll be mocking this request too in the mock server and we'd like to make sure a user can get in via the link, are we just deferring this until that's setup to avoid the trouble right now?

Reply:
Yes, I'm waiting to complete this test until the mock server is set up, since it would take a significant workaround right now to make it work (logging in to email via Safari). My plan is to stop the test here for now and then complete the flow once we have the mock server set up.

I'll add a note in the test as a reminder that it's not a complete flow yet.